graph.go

```diff
@@ -195,16 +195,21 @@ func (c *GraphCmd) Execute(args []string) error {
 }
 
 func relPath(base, path string) string {
-	rp, err := filepath.Rel(base, path)
+	pathWithoutSymlinks, err := filepath.EvalSymlinks(path)
```

Comment:
Yeah, it looks like it's needed to be able to calculate relative paths/prefixes in an expected way.